server/pkg/builtin/manifest_ja.yml

```diff
@@ -11,6 +11,9 @@ extensions:
           camera:
             title: カメラ初期位置
             description: ページロード後最初に表示するカメラの位置を設定します。
+          allowEnterGround:
+            title: ホゲホゲホゲ
+            description: ホゲホゲホゲホゲホゲホゲホゲホゲ。
```

Comment:
```suggestion
          allowEnterGround:
            title: 地下へ入る
            description: Onにすることでカメラが地表面へ潜り込むようになります。
```

Reply:
Kindly check the translation! 

@lavalse Plz check if these definitions should come here or otherwhere else.